docs/getting-started/key-components.md

```diff
@@ -1,25 +1,45 @@
 # Key Components
 
+Before you start automating workflows with n8n, it's important to understand some key components and concepts.
+
+## Editor
+
+The [Editor](../reference/glossary.md#editor-ui) is the web interface used to create [workflows](#Workflow). It is accessed through a web browser at a designated website address.
+
+![Editor](images/Editor.png)
```

Comment:
All the images on this page are broken and some of the links are broken as well.